### PR TITLE
devops: Don't auto-install brew deps

### PR DESCRIPTION
**Pull request by max-sixty**
As discussed in https://github.com/PRQL/prql/pull/3333#issuecomment-1678374434

(need to have another look in the morning)

### CODE REVIEW

File: Taskfile.yml [hunk 1]
```diff
@@ -26,6 +26,12 @@ vars:
     bacon cargo-audit cargo-insta cargo-release default-target mdbook
     mdbook-admonish mdbook-footnote mdbook-toc wasm-bindgen-cli wasm-opt
     wasm-pack
+  brew_dependencies: |
+    brew
+    hugo
+    npm
+    python3
+    elixir
```
Comment (eitsupi) [inline Taskfile.yml]: I'm wondering if it's worth including elixir here.
I don't think prql-elixir has been substantially edited in the last few months.

Comment (max-sixty) [reply Taskfile.yml]: Yes OK

**Comment by max-sixty** [comment]
Having some trouble with the devcontainer...

This last passed two weeks ago.

I know you're keen on not running this too often, but I do wonder whether running this nightly would help us understand the cause when we get a problem like this — is it the parent image? Is it `cargo binstall`? I'm not sure the best way to figure this out...

---

I added a fallback to installing from source [here](https://github.com/PRQL/prql/pull/3346/files#diff-cd2d359855d0301ce190f1ec3b4c572ea690c83747f6df61c9340720e3d2425eR109), but now the binaries aren't installed to the `PATH`: https://github.com/PRQL/prql/actions/runs/5870585733/job/15917918066?pr=3346#step:6:1016

File: Taskfile.yml [hunk 2]
```diff
@@ -22,17 +22,34 @@ vars:
...
+    cargo-release
+    default-target
+    mdbook
+    mdbook-admonish
+    mdbook-footnote
+    mdbook-toc
+    wasm-bindgen-cli
+    wasm-opt
     wasm-pack
+  # Excluding `elixir` atm given it's not enabled on Mac and not yes supported
```
Comment (max-sixty) [inline Taskfile.yml]:
```suggestion
  # Excluding `elixir` atm given it's not enabled on Mac and currently unsupported
```

File: Taskfile.yml [hunk 3]
```diff
@@ -22,17 +22,34 @@ vars:
...
+    python3
 
 tasks:
   # main installer is "setup-dev" which calls other tasks
   setup-dev:
     preconditions:
       - sh: which clang
         msg: |
+
+
```
Comment (max-sixty) [inline Taskfile.yml]:
```suggestion
```

**Comment by max-sixty** [comment]
Merging this despite the devcontainer failure — it's unrelated.

**Comment by max-sixty** [comment]
> is it the parent image? Is it `cargo binstall`? I'm not sure the best way to figure this out...

Looks like that issue is related to https://github.com/cargo-bins/cargo-binstall/issues/1292

**Pull request merged**